services/docker.go

```diff
@@ -44,6 +45,17 @@ func CreateNetwork(name string) error {
 
 	return nil
 }
+func ConnectNetwork(containerId, networkId string) error {
+	err := c.NetworkConnect(context.Background(), networkId, containerId, &network.EndpointSettings{})
+
```

Comment:
Yes because PWD container needs to be in all networks to be able to route the request.